Immense.RemoteControl.Server/Abstractions/IHubEventHandler.cs

```diff
@@ -1,6 +1,7 @@
 using Immense.RemoteControl.Server.Models;
 using Immense.RemoteControl.Shared;
 using Immense.RemoteControl.Shared.Enums;
+using Immense.RemoteControl.Server.Services;
 
 namespace Immense.RemoteControl.Server.Abstractions;
 
```

Comment:
Should we add some docs here indicating that this service is added to SC in the RemoteControlServerBuilder with singleton lifetime?

Reply:
Good point.  Added that info to the interfaces and to `RemoteControlServerBuilder `.